### PR TITLE
Add strip functionality to registers for bitfields

### PR DESCRIPTION
**Pull request by MabezDev**
Mirroring capability of peripherals for registers.

### CODE REVIEW

File: svdtools/patch.py [hunk 1]
```diff
@@ -705,6 +710,29 @@ def iter_fields(self, fspec):
             if matchname(name, fspec):
                 yield ftag
 
+    def strip(self, substr, strip_end=False):
```
Comment (nickray) [inline svdtools/patch.py]: Maybe `lstrip` and `rstrip` would be nicer symmetry? I haven't personally used `strip` though and would be breaking.

Comment (nickray) [inline svdtools/patch.py]: Python stdlib `strip` has different behaviour, and I'm not a fan of the asymmetry. Maybe `remove_prefix` and `remove_postfix` (or `_start` and `_end`) would have been nicer :man_shrugging:

**Comment by nickray** [comment]
Released version is controlled via `svdtools/VERSION`, feel free to edit too in the future if you plan to iterate often.

**Comment by MabezDev** [comment]
Thanks!

> Released version is controlled via svdtools/VERSION

I'll keep that in mind for next time :)

**Pull request merged**